ironfish/src/rpc/types.ts

```diff
@@ -54,3 +56,59 @@ export const RpcEncryptedNoteSchema: yup.ObjectSchema<RpcEncryptedNote> = yup
     serialized: yup.string().defined(),
   })
   .defined()
+
+export type RpcBlockHeader = {
+  hash: string
+  /**
+   * @deprecated Please use previousBlockHash instead
+   */
+  previous: string
+  sequence: number
+  previousBlockHash: string
+  difficulty: string
+  noteCommitment: string
+  transactionCommitment: string
+  target: string
+  randomness: string
+  timestamp: number
+  graffiti: string
+  work: string
+  noteSize?: number
+}
```

Comment:
would love your thoughts on which one of these should be deprecated. we can deprecate later also